haaska.py

```diff
@@ -144,6 +144,8 @@ def mk_appliance(x):
             o['friendlyName'] = x['attributes']['friendly_name']
             if entity_domain(x) == 'scene':
                 o['friendlyName'] += ' Scene'
+            elif entity_domain(x) == 'group':
```

Comment:
I'm still unsure how much I really like appending "scene" or "group" to the name, but it does seem like the easiest way to avoid collisions and make the verbal interface a bit more natural.